### ISSUE TITLE
Graph

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rmanor/currentwidget#5]
```
time/current graph view.

```

Original issue reported on code.google.com by `ran.ma...@gmail.com` on 26 Aug 2010 at 2:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `ran.ma...@gmail.com` on 26 Dec 2010 at 10:50
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
graph view released in v0.22.
download from Market.

```

Original comment by `ran.ma...@gmail.com` on 9 Jan 2011 at 9:04
- Changed state: **Verified**
